packages/embeds/embed-react/playwright/tests/basic.e2e.ts

```diff
@@ -46,7 +46,7 @@ test.describe("React Embed", () => {
 
   // TODO: This test is extremely flaky and has been failing a lot, blocking many PRs. Fix this.
   // eslint-disable-next-line playwright/no-skipped-test
-  test.describe.skip("Element Click Popup", () => {
+  test.describe("Element Click Popup", () => {
```

Comment:
Couldn't get this to fail locally, will keep an eye out to see if it fails on ci

Reply:
Okay so this is definitely failing, investigating

Reply:
Fixed